vipps-recurring-api-faq.md

```diff
@@ -94,9 +95,79 @@ change to Vipps in different ways:
 * Paper invoice: Include a QR code with unique URL for the customer, same as
   above.
 
+## How can I move agreements between merchants and sale units?
+
+Merchants sometimes need to move customer agreements from one merchant
+to another, or from one sale unit to another.
+
+Clarification of terms:
+* Merchant: A juridical unit, typically called a business or company, identified with organization number ("orgno").
+* Sale unit: A merchant can have one or more sale units. It may be different
+  brands, different physical locations, different services, etc.
+* MSN: The unique id of a sale unit.
+  MSN is short for "Merchant Serial Number", but (sadly) identifies a sale unit, not a merchant.
+
+First of all: A merchant very rarely "changes organization number".
+That only occurs in special cases.
+An organization number uniquely identifies a juridical unit and cannot usually be changed.
+
+Typical cases:
+
+1. If the merchant wants to move agreements from
+   one MSN to another,
+   and both MSNs are under the same orgno,
+   Vipps may be able to help.
+2. If the merchant wants to move agreements from
+   one orgno to another,
+   and both orgnos are owned by the same parent orgno,
+   Vipps may be able to help.
+3. If the merchant wants to move agreements from
+   one orgno to another,
+   and the orgnos are not owned by a parent company,
+   Vipps may be able to help.
+   The users have entered agreements with an orgno, and Vipps can not
+   automatically move the agreements without the user's consent to enter
+   agreements with the new orgno.
+   The users may need to enter new agreements with the new orgno.
+4. If the merchant wants to move agreements between two independent orgnos
+   (not owned by a parent orgno),
+   splits or fissions out a new orgno,
+   is merged with another orgno,
+   is acquired by another orgno,
+   etc:
+   Vipps may *_not_* be able to help.
+   The users have entered agreements with an orgno, and Vipps can not
+   automatically move the agreements without the user's consent to enter
+   agreements with the new orgno.
+   The users may need to enter new agreements with the new orgno.
+
+The general process is:
+- Log in on
+  [portal.vipps.no](https://portal.vipps.no)
+  and enter a new agreement with Vipps for the new orgno.
+- Order "Vipps Faste betalinger" for the new orno. on
+  [portal.vipps.no](https://portal.vipps.no).
+- Inform all existing customers of the new orgno.
+- Send a confirmation to avtale@vipps.no that all customers have been informed,
+  including a copy of the information sent.
+- Contact Vipps (your KAM or partner manager) to plan the switch, and
+  the date and time for moving the agreements.
```

Comment:
I think maybe the order should be different here, since they should try to inform their users about the time of the migration, which necessitates that they have already organized the migration with vipps?

Reply:
Hmm, good point. Will tweak.